src/libraries/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs

```diff
@@ -775,6 +775,26 @@ public sealed partial class OptionalAttribute : System.Attribute
     {
         public OptionalAttribute() { }
     }
+    public enum PosixSignal
```

Comment:
I've manually added these to the `ref` file, I don't know if there is an automated way.

Reply:
The automated way is documented in https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/updating-ref-source.md . It would be a good idea to rerun the automated process to make sure that the formatting is right.

Reply:
But beware, it currently removes attributes...

Reply:
I ran it and copied in the `PosixSignal` bits.
I'm surprised to see `internal` instead of `private` on the `PosixSignalRegistration` constructor.